.github/workflows/ci.yml

```diff
@@ -93,3 +93,70 @@ jobs:
       - name: Compile
         working-directory: build
         run: make -j4
+  windows-build:
+    runs-on: windows-2019
+    name: Windows CI
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup MSYS2
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: MINGW64
+        update: true
+        install: >-
+          git
+          make
+        pacboy: >-
+          toolchain:p
+          cmake:p
+          ninja:p
+          libusb:p
+          boost:p
+          qt5:p
+          spdlog:p
+          volk:p
+          libsndfile:p
+          soapysdr:p
+          fftw:p
+          rtl-sdr:p
+    - name: Clone and Build GNU Radio
+      shell: msys2 {0}
+      run: |
+        git clone --recursive https://github.com/cozycactus/gnuradio.git
```

Comment:
I think I'll hold off merging this PR until https://github.com/gnuradio/gnuradio/pull/6971 is merged. That will let us use the upstream GNU Radio.

Reply:
We should be able to switch this to upstream now.

I think `--recursive` can be removed as well, since GNU Radio doesn't have submodules anymore.